src/index-generator.ts

```diff
@@ -0,0 +1,263 @@
+            padding: 6px 8px;
+            cursor: pointer;
+            border-radius: 4px;
+        }
+
+        .forum-header:hover {
+            background-color: var(--background-modifier-hover);
+            color: var(--channel-text-hover);
+        }
+
+        .forum-title {
+            display: flex;
+            align-items: center;
+        }
+
+        .forum-icon {
+            margin-right: 6px;
+        }
+    </style>
+</head>
+<body>
+    <div class="app">
+        <div class="guild-header">
+            <h1>${guildName} Discord Transcripts</h1>
+        </div>
+
+        <div class="main-content">
+            <div class="channels-container">
+                <div class="channel-category">
+                    <div class="category-name">Channels</div>
+                    <ul class="channels-list">
+                        ${regularChannels.map(channel => `
+                        <li class="channel-item">
+                            <span class="channel-hash">#</span>
+                            <a href="./${sanitizeFilename(channel.filename)}">${channel.displayName.replace('#', '')}</a>
+                        </li>
+                        `).join('')}
+                    </ul>
+                </div>
+
+                ${Object.entries(groupedTranscripts).map(([forumName, threads]) => `
+                <div class="channel-category">
+                    <div class="forum-header">
+                        <div class="forum-title">
+                            <span class="forum-icon">📋</span>
+                            ${forumName}
+                        </div>
+                    </div>
+                    <ul class="channels-list forum-channels">
+                        ${threads
+                            .sort((a, b) => {
+                                // Sort by lastActivity timestamp (most recent first)
+                                // If no timestamp available, put at the bottom
+                                if (!a.lastActivity) return 1;
+                                if (!b.lastActivity) return -1;
+                                return new Date(b.lastActivity).getTime() - new Date(a.lastActivity).getTime();
+                            })
+                            .map(thread => `
+                        <li class="channel-item">
+                            <span class="channel-hash">#</span>
+                            <a href="./${sanitizeFilename(thread.filename)}">${thread.displayName.split('/')[1]}</a>
+                            ${thread.lastActivity ? `<span class="timestamp" data-timestamp="${new Date(thread.lastActivity).toISOString()}"></span>` : ''}
+                        </li>
+                        `).join('')}
+                    </ul>
+                </div>
+                `).join('')}
+            </div>
+        </div>
+        <div><span class="timestamp" data-timestamp="${new Date().toLocaleString()}"> Generated: </span></div>
```

Comment:
Add a "powered by discord-html-transcripts". We removed it from all the individual pages, but it's just fair to have it on the main page.